src/widgets/capture/capturewidget.cpp

```diff
@@ -490,6 +490,7 @@ void CaptureWidget::paintEvent(QPaintEvent* paintEvent)
     if (m_activeTool && m_mouseIsClicked) {
         painter.save();
         m_activeTool->process(painter, m_context.screenshot);
+        update();
```

Comment:
If I don't add update() here, the painting outside the selection rect will not flush until mouse move through.I'm new to qt5 and don't sure it will influence performance.
![2022-04-06_12-01](https://user-images.githubusercontent.com/49367127/161893268-7f447b54-7729-42c3-9f65-dfa2d49f7744.png)